backend/django/core/templates/projects/create/create_wizard_labels.html

```diff
@@ -73,15 +88,59 @@ <h3>Instructions</h3>
 {% endblock %}
 
 {% block scripts_body %}
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-csv/0.71/jquery.csv-0.71.min.js"></script>
 <script type="text/javascript">
-$('.formset_row_label').formset({
-  addText: 'add label',
-  deleteText: 'remove label',
-  prefix: 'label_set',
-  added: function() {
-    // Reset drop down and errors from both columns when adding new rows
-    $('tr.formset_row_label').last().children(':first').find('ul.errorlist li').empty();
+  function handleUpload(event) {
```

Comment:
Found a new issue with adding a lot of labels which we'll need to resolve before merging in. I tried to add a file with thousands of labels. It added them correctly to the table, but upon submitting I got this error 

<img width="929" alt="Screen Shot 2021-08-02 at 5 10 52 PM" src="https://user-images.githubusercontent.com/10951569/127924841-ab228b9c-ff5b-4ac8-80aa-695b40ec50a2.png">

I suspect this is an error which could be solved by updating a constant in `backend/django/smart/settings.py`. See this post for reference https://stackoverflow.com/questions/47585583/the-number-of-get-post-parameters-exceeded-settings-data-upload-max-number-field